apps/cookbook/src/app/showcase/list-virtual-scroll-showcase/list-virtual-scroll-showcase.component.html

```diff
@@ -10,36 +16,49 @@
     <a href="https://github.com/dhilt/ngx-ui-scroll" target="_blank">ngx-ui-scroll library</a>.
   </p>
 </div>
-<div class="example-wrapper">
-  <h2>With preloaded items</h2>
-  <p>100 items in the list, but approximatly 25 items rendered on the page at a time.</p>
-  <cookbook-code-viewer [html]="itemsHtml"></cookbook-code-viewer>
-  <cookbook-iphone
-    src="/examples/list-virtual-scroll/with-items"
-    showExternalLink="true"
-  ></cookbook-iphone>
+<div class="page-example">
```

Comment:
Seems naming was copy-pasted from Page example :)
```suggestion
<div class="example-wrapper">
```